Cargo.toml

```diff
@@ -41,14 +41,17 @@ tokio = { version = "1.35.1", features = [
     "net",
     "time",
     "macros",
+    "signal",
     "rt-multi-thread",
 ] }
 tokio-rustls = { version = "0.25.0", optional = true }
+tracing = "0.1"
```

Comment:
@jonhoo I saw them putting it under feature flag and combining with `log`, e.g. [`here`](https://github.com/fede1024/rust-rdkafka/blob/master/src/log.rs). But my understanding is that this has grown into a standard and can be included without opt-in. Those subscribing will be able to opt-out of the events originated from the crate free of CPU/mem charge, wdyt?